packages/react-components/tsconfig.json

```diff
@@ -1,22 +1,17 @@
 {
+  "extends": "../../tsconfig.base.json",
+  "include": ["src"],
   "compilerOptions": {
-    "baseUrl": ".",
+    "target": "ES5",
+    "module": "CommonJS",
+    "lib": ["ES2015", "dom"],
```

Comment:
story files have been checked by different tsconfig when they lived within react-examples
> react-examples used 2 ts configs which were using different bases.

Now we invoke `tsc` the same way across all vNext.  we use es2015 lang features in story files/helpers thus we need to temporarily set lib to es2015. migration generator contains updated ts configs, which use `es2015` also for `target` -> this will be applied in future PR